.gitignore

```diff
@@ -3,3 +3,7 @@
 cache
 run
 verify
+
+# Foundry setup for VS Code users
+remappings.txt
```

Comment:
`remappings` defined in foundry.toml are the CI's source of truth and so not setting them will fail the CI, especially since it currently doesn't remap to "@openzeppellin" as is done here:

- https://github.com/rugpullindex/ERC4973/pull/8/files#diff-a9c109dc3fa60963707e32ea2ac8eebc3cdd592f99f78a4c72457e59a0644008R4

Make sure to always use remappings from foundry.toml file:

https://github.com/rugpullindex/ERC4973/blob/070fa229e11d898f6cb666ae2e4b2d6478f03d5c/foundry.toml#L5